specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2023-08-01/clusters.json

```diff
@@ -0,0 +1,1088 @@
+          "format": "date-time",
+          "readOnly": true
+        },
+        "serviceEndpoint": {
+          "description": "Region specific DataPath Endpoint of the cluster.",
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceProviderObjectId": {
+          "description": "Object id of RP Service Principal",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ClusterPatchProperties": {
+      "description": "Cluster properties.",
+      "type": "object",
+      "properties": {
+        "cloudManagementEndpoint": {
+          "description": "Endpoint configured for management from the Azure portal",
+          "type": "string"
+        },
+        "aadClientId": {
+          "description": "App id of cluster AAD identity.",
+          "type": "string"
+        },
+        "aadTenantId": {
+          "description": "Tenant id of cluster AAD identity.",
+          "type": "string"
+        },
+        "desiredProperties": {
+          "description": "Desired properties of the cluster.",
+          "$ref": "#/definitions/ClusterDesiredProperties"
+        }
+      }
+    },
+    "SoftwareAssuranceProperties": {
+      "description": "Software Assurance properties of the cluster.",
+      "type": "object",
+      "properties": {
+        "softwareAssuranceStatus": {
+          "description": "Status of the Software Assurance for the cluster.",
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "SoftwareAssuranceStatus",
+            "modelAsString": true
+          },
+          "readOnly": true
+        },
+        "softwareAssuranceIntent": {
+          "description": "Customer Intent for Software Assurance Benefit.",
+          "type": "string",
+          "enum": [
+            "Enable",
+            "Disable"
+          ],
+          "x-ms-enum": {
+            "name": "SoftwareAssuranceIntent",
+            "modelAsString": true
+          }
+        },
+        "lastUpdated": {
+          "description": "TimeStamp denoting the latest SA benefit applicability is validated.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      }
+    },
+    "IsolatedVmAttestationConfiguration": {
+      "description": "Attestation configurations for isolated VM (e.g. TVM, CVM) of the cluster.",
+      "type": "object",
+      "properties": {
+        "attestationResourceId": {
+          "description": "Fully qualified Arm Resource Id of the Microsoft Azure Attestation resource associated with this cluster.",
+          "type": "string",
+          "format": "arm-id",
+          "x-ms-arm-id-details": {
+            "allowedResources": [
+              {
+                "type": "Microsoft.Attestation/attestationProviders",
+                "scopes": "/"
+              }
+            ]
+          },
```

Comment:
`scopes` here should be an array
https://azure.github.io/autorest/extensions/#x-ms-arm-id-details